doc/admin-guide/introduction.en.rst

```diff
@@ -187,14 +187,6 @@ requests and manage, control, and monitor the health of the system.
    first-served order. This connection queueing shields users from any
    server restart downtime.
 
-The figure below illustrates the |TS| processes.
```

Comment:
Rather than removing all this, I think the picture needs to be modified / replaced with one that shows the current process layout.

Reply:
For two processes? The one before just showed the three (which included traffic_cop) in a circle with arrows pointing to each other. I'm not sure a picture here adds much. At least not one like we had before.

Reply:
Well, if we don't want to change the picture, we should at least "rm" process.jpg as Bryan said.

Reply:
Sounds good. I removed the image file in my earlier push.

Thanks for the review.